test/functional/legacy/089_findfile_spec.lua

```diff
@@ -0,0 +1,30 @@
+-- - Some tests for findfile() function
+
+local helpers = require('test.functional.helpers')
+local feed = helpers.feed
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('findfile', function()
```

Comment:
Good thought about separating the findfile tests, but I think it's enough to just have them in separate `describe(...)` blocks in the same `089_foo_spec.lua` file. Of course, for non-legacy tests, we wouldn't have unrelated tests in the same file, but in the case of legacy tests it's less confusing to keep test 123 in a single 123 file.